src/main/java/jnr/ffi/Struct.java

```diff
@@ -239,6 +245,34 @@ public double doubleValue() {
         }
     }
 
+    public static final class Alignment extends Number {
```

Comment:
Ahh, I see. I wonder if this was an attempt at improving type-safety. I don't really see the value in these Number subtypes that just wrap int, but we'll leave it for now.